<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogOn</class>
 <widget class="QWidget" name="LogOn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>100</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/head1.jpg);
border-width:1px;
border-radius:36px;
border-style:solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>96</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-radius:40px;
border-style:solid;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>190</y>
     <width>300</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-top-color:rgba(255,255,255,0);
border-right-color:rgba(255,255,255,0);
border-left-color:rgba(255,255,255,0);
border-bottom-color:rgb(129,129,129);</string>
   </property>
   <property name="text">
    <string>zjgm2201</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>230</y>
     <width>300</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-top-color:rgba(255,255,255,0);
border-right-color:rgba(255,255,255,0);
border-left-color:rgba(255,255,255,0);
border-bottom-color:rgb(129,129,129);</string>
   </property>
   <property name="text">
    <string>0123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logon">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>300</y>
     <width>300</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/logon.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="regster">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(176, 176, 176);
backgrund:transparent;
border:0px;</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setUp">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="severip">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>365</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-top-color:rgba(255,255,255,0);
border-right-color:rgba(255,255,255,0);
border-left-color:rgba(255,255,255,0);
border-bottom-color:rgb(129,129,129);</string>
   </property>
   <property name="text">
    <string>172.19.29.82</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器ip:</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>logon</zorder>
  <zorder>regster</zorder>
  <zorder>setUp</zorder>
  <zorder>severip</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
